parquet-variant/src/decoder.rs

```diff
@@ -98,7 +98,7 @@ impl TryFrom<u8> for VariantPrimitiveType {
     }
 }
 
-/// The number of bytes used to store offsets in the [`VariantMetadataHeader`]
+/// The number of bytes used to store offsets in the `VariantMetadataHeader`
```

Comment:
This is not quite accurate. Suggest this instead?
```suggestion
/// Used to unpack offset array entries such as metadata dictionary offsets or object/array value
/// offsets. Also used to unpack object field ids. These are always derived from a two-bit
/// `XXX_size_minus_one` field in the corresponding header byte.
```